src/client/app/effects/github.ts

```diff
@@ -14,6 +14,7 @@ import * as moment from 'moment';
 import * as fromRoot from '../reducers';
 
 const FileSaver = require('file-saver');
+const UNKNOWN_GIST_OWNER_ID = '<unknown'; // GitHub IDs can only have alphanumeric characters
```

Comment:
I assume you want a closing ">" as well?  Also, not sure what you mean by your comment.  Is it that you're including the brackets so as to make 100% sure that no user will ever have this name?  If so, makes sense, but maybe change the comment to "intentionally using brackets that are not allowed in a GitHub ID (which only allows alphanumeric characters)"